[OV20] NV12toRGB and NV12toBGR operations specification (#7595)
openvinotoolkit/openvino: docs/ops/image/NV12toBGR_8.md

## NV12toBGR <a name="NV12toBGR"></a> {#openvino_docs_ops_image_NV12toBGR_8}

**Versioned name**: *NV12toBGR-8*

**Category**: *Image processing*

> **Short description**: *NV12toBGR* performs image conversion from NV12 format to BGR.

Reviewer: ```suggestion
**Short description**: *NV12toBGR* performs image conversion from NV12 to BGR format.
```

**Detailed description**:

Similar to *NV12toRGB* but output channels for each pixel are reversed so that: first channel is `blue`, second one is `green`, last one is `red`.  See detailed conversion formulas at [NV12toRGB](NV12toRGB_8.md)

**Inputs:**

> Same as specified for [NV12toRGB](NV12toRGB_8.md) operation.

Reviewer 2: does this link work?

Author: I tried it locally, it worked well. In Azure it seems that upload of generated docs is disabled
![image](https://user-images.githubusercontent.com/79140263/134522631-c02b0f4e-7187-4510-8c5a-b3c2d2eb49e1.png)

**Outputs:**

> * **1**: A tensor of type *T* representing converted image in BGR format. Dimensions:

Reviewer: ```suggestion
* **1**: A tensor of type *T* representing an image converted in BGR format. Dimensions:
```

  * `N` - batch dimension

>   * `H` - height dimension is same as image height

Reviewer: ```suggestion
  * `H` - height dimension is the same as the image height
```

>   * `W` - width dimension is same as image width

Reviewer: ```suggestion
  * `W` - width dimension is the same as the image width
```

>   * `C` - channels dimension equals to 3. First channel is Blue, second one is Green, last one is Red

Reviewer: ```suggestion
  * `C` - channels dimension is equal to 3. The first channel is Blue, the second one is Green, the last one is Red
```

**Types:**

> * *T*: `uint8` or `float32` type.

Reviewer 3: Shouldn't `float32` be replaced with `any supported floating-point type`?

Author: @ilya-lavrenov can you comment on this? Technically any type except 'bool', 'int4', 'int8' is fine to store range 0-255

Reviewer 4: I'm OK with it, but will plugins support it?

@jane-intel what is the policy for such cases?

Reviewer 2: *T*: `uint8` or any supported floating-point type.

Author: Thanks, corrected
